requirements/app/base.txt

```diff
@@ -7,3 +7,4 @@ s3fs>=2022.5.0, <=2022.7.1
 croniter>=1.3.0, <1.4.0  # strict; TODO: for now until we find something more robust.
 traitlets<5.2.0 # Traitlets 5.2.X fails: https://github.com/ipython/traitlets/issues/741
 arrow>=1.2.0, <=1.2.2
+lightning_utilities>=0.2.0
```

Comment:
I thought it will be called lightning-tools. Also the underscore is a typo, right? I see it here: https://pypi.org/project/lightning-utilities/

```suggestion
lightning-utilities>=0.2.0
```

Reply:
Jirka chose utilities in the end. Underscore vs dash does not matter, but I'll commit